README.md

```diff
@@ -46,6 +54,10 @@ The Fornax Science Console is a compute system in the cloud near to NASA data in
      * if it is a large amount of data, consider creating a zip or tar archive first
   * Home Directory
     * When you log into the science console, the active directory is your $HOME directory.  This directory is unique to you: edits and uploads are not visible to other users.  
+        * Raen thinks there are directories in $HOME that are shared (e.g., `efs` and `s3`), and perhaps just mirrored or symlinked into $HOME. It would be nice to get clarification. Specifically:
+            * Which directories are/aren't shared?
+            * Which directories does the user have write access to, and are there any restrictions/considerations for putting stuff there (other than disk size)?
+            * Which directories can/should the user look in to discover data they have access to? (e.g., `s3` has various archive data, `efs` has some data shared by users, anything else?)
```

Comment:
The $HOME directory is a bit confusing to me. I added some notes here with relevant questions.